Solutions/Synack/SolutionMetadata.json

```diff
@@ -0,0 +1,14 @@
+{
+	"publisherId": "synack",
+	"offerId": "synack",
+    "providers": ["Microsoft"],
+	"categories": {
+		"domains" : ["Security - Others"]
+	},
+	"support": {
+        "tier": "Microsoft",
```

Comment:
tier needs to be partner
support needs to be Synack so please remove Microsoft from support. This is a new pattern of solutions so we are helping by just building it and not publishing.